webapp/views/App/views/Analysis/Chain/Chain.js

```diff
@@ -0,0 +1,59 @@
+import './Chain.scss'
+
+import React from 'react'
+
+import * as Validation from '@core/validation/validation'
+
+import * as Chain from '@common/analysis/processingChain'
+
+import LabelsEditor from '@webapp/components/survey/LabelsEditor'
+import CyclesSelector from '@webapp/components/survey/CyclesSelector'
+
+import { useI18n } from '@webapp/store/system'
+
+import { useChain } from './store'
+
+const ChainComponent = () => {
+  const i18n = useI18n()
+  const { chain, dirty, onUpdate, onSave, onDismiss } = useChain()
+  const validation = Chain.getValidation(chain)
+
+  return (
```

Comment:
this component is not like the current one...
ButtonRStudio is missing and I would extract the ButtonBar as it was before

Reply:
I did both things into the next branch, should I do into this too?